Makefile

```diff
@@ -19,6 +19,7 @@ export PYTHONPATH = $(shell echo "$$PYTHONPATH"):$(shell python -c 'import os; p
 all: clean-pyc check test
 
 check:
+	@$(PYTHON) scripts/check_crlf.py pygments build external || true
```

Comment:
`|| true` can be omitted since this shouldn't complain...

Reply:
I'm not familiar with that Makefile mechanism to start with, will remove it :)

Reply:
It's shell syntax - `|| true` will ignore the exit status of the script and exit with success.  It's used for the other invocations because they are (at the moment) producing errors.

Once you fix all the warnings you can remove this for all the other lines so that they actually do complain if something is wrong :)

Reply:
Ah, thanks, that makes sense!